docs/markdown/concepts-and-architecture/channels.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: Channels
+nav_order: 2
+permalink: /concepts/channels
+parent: Concepts and Architecture
+---
+# Channels
+A channel is a concept used as part of the bridge which facilitates the delivery of multiple RPCs in a single direction. A channel consists of a sender and a receiver, each being a piece of business logic that runs on opposite chains. Any user or system wanting to send a message across the bridge must submit their RPC to the channel. Channels at the very least are used to provide some deliverability guarantees to a RPC message, and to provide replay protection across multiple messages.
+
+## Channel Qualities
+There are many different designs and ideas for how channels can be implemented, each with different tradeoffs and qualities. This page explores some of these and attempts to model out the domain of channel design such that we can eventually pick the design that suits our bridge best.
+
+## Guaranteed Deliverability vs Guaranteed Delivery
+As described before, a channel must ensure deliverability, but it does not need to ensure delivery. Guranteed delivery is great for users and UX though, but complicated to get right and dependent on various assumptions. More complex channels can target both.
```

Comment:
Instead of "As described before" -> "As described \[here\](/concepts/components#deliverability-and-delivery)"